Fixed a typo in Specify How Fonts Degrade lesson.

turnonsync opened the pull request:
`-->!` inside the tests section is misleading and prompts a learner
to put the exclamation mark inside the editor when it is not needed as `-->!` is
also followed by a period. This is telling the user to close an HTML comment.

Instead of "`-->!`.", it should be "`-->`.".

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [ x] None of my changes are plagiarized from another source without proper attribution.
- [ x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [ x] My changes do not use shortened URLs or affiliate links.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

turnonsync committed:
Fixed a typo in Specify How Fonts Degrade.

'-->!' inside the code is misleading and prompts a learner
to put the exclamation mark when it is not needed as '-->!' is
also followed by a period.

Instead of "'-->!'.", it should be "'-->'."

RandellDawson commented:
Thank you for opening this pull request.

This is a standard message notifying you that we’ve reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

**Reason:**  The current curriculum is for JavaScript not Python.

Thank you and happy coding.

RandellDawson commented on curriculum/challenges/english/01-responsive-web-design/basic-css/specify-how-fonts-should-degrade.english.md:
```diff
@@ -39,7 +39,7 @@ tests:
     testString: assert(/\s*h2\s*\{\s*font-family\:\s*(\'|")?Lobster(\'|")?,\s*monospace\s*;\s*\}/gi.test(code));
   - text: Comment out your call to Google for the <code>Lobster</code> font by putting <code>&#60;!--</code> in front of it.
```
```suggestion
  - text: You should comment out your call to Google for the <code>Lobster</code> font by putting <code>&#60;!--</code> in front of it.
```

turnonsync replied:
I have committed the new change.

RandellDawson commented on curriculum/challenges/english/01-responsive-web-design/basic-css/specify-how-fonts-should-degrade.english.md:
```diff
@@ -39,7 +39,7 @@ tests:
   - text: Comment out your call to Google for the <code>Lobster</code> font by putting <code>&#60;!--</code> in front of it.
     testString: assert(new RegExp("<!--[^fc]", "gi").test(code));
-  - text: Be sure to close your comment by adding <code>--&#62;!</code>.
+  - text: Be sure to close your comment by adding <code>--&#62;</code>.
```
```suggestion
  - text: You should close your comment by adding <code>--&#62;</code>.
```

turnonsync replied:
Done.

RandellDawson reviewed (changes requested):
If we are going to make changes to the `text` wording, let's make it consistent with the other tests by using `should`.

turnonsync committed:
Made text wording consistent for the test instructions.

RandellDawson commented:
Reviewers/Mods:  I have an earlier PR which makes some of the changes of this PR, but it is still in Draft mode.  Go ahead an merge this one first and I will fix any merge conflicts on my PR.

turnonsync committed:
Changed "To begin with," to "To begin," and fixed a minor typo.

If we are going to use "To begin with," the sentence should be
"To begin with the lesson/challenge", otherwise, "To begin" will
be more than enough.

Also added a space between "font(using HTML..." and changed it to
"font (using HTML...".

RandellDawson commented:
@turnonsync Just saw your new changes.  I am still fine with them.

Manish-Giri reviewed (approved):
LGTM. Thank You for your contribution.
